ActiasRuntimeTools/main.cpp

```diff
@@ -25,8 +26,60 @@ typedef ExecutableParseError ACTIAS_ABI ActiasLoadNativeExecutableProc(INativeEx
 
 typedef void ACTIAS_ABI ActiasBuildExecutableProc(IBlob** ppExecutableData, const ActiasExecutableBuildInfo* pBuildInfo);
 
+void build(const String& executable, int& result)
+{
+    LibraryLoader loader("ActiasSDK", true);
+
+    auto ExecutableRead = File::ReadAllBytes(executable);
```

Comment:
```suggestion
    auto executableRead = File::ReadAllBytes(executable);
```
Возможно `const`, хотя не уверен, что будет компилироваться.

Reply:
Компилируется